src/main/scala/com/fulcrumgenomics/bam/FilterBam.scala

```diff
@@ -87,6 +89,7 @@ class FilterBam
     val in       = SamSource(input)
```

Comment:
I noticed on line 88 we have an unused `ProgressLogger`! Mind deleting, or wiring it in?